subprojects/docs/src/docs/stylesheets/userGuideHtml.xsl

```diff
@@ -184,6 +184,7 @@
                                         <ul id="dependency-management-glossary">
                                             <li><a href="/userguide/dependency_types.html">Dependency Types</a></li>
                                             <li><a href="/userguide/repository_types.html">Repository Types</a></li>
+                                            <li><a href="/userguide/dependency_cache.html">Dependency Cache</a></li>
```

Comment:
Hmm... I had always imagined the Glossary as a sort of dictionary where one could find answers to:
* What is a configuration?
* What is a repository?
* What is variant-aware dependency management?

I'd say the inclusion of "Dependency Cache" also makes it hard for me to know what pieces of docs belong or don't belong in the Glossary. Could we flesh out more of the expected end vision here?

Reply:
Reading more below, I'm not sure where the Dependency Cache chapter should exist, if at all.

Reply:
> Hmm... I had always imagined the Glossary as a sort of dictionary where one could find answers to:

What you describe is pretty much this: https://github.com/gradle/gradle/pull/4082. A short reference on terminology and where I can find additional information.

The existing content in the glossary is reference material from my perspective. As a user I don't need to see all types of dependencies or repositories I can declare in the dependency management section. I want to know how I can do it and if I want to explore more then I can go to the reference. But overall there's no need to convolute the user manual with every possible scenario.

> I'd say the inclusion of "Dependency Cache" also makes it hard for me to know what pieces of docs belong or don't belong in the Glossary.

I felt conflicted about that content in general. As it is right now it's too long to be included in the "introduction". It also has marketing aspects in the content which I feel do not necessarily need to live there. More on this below. Overall, I felt like it could be part of the "introduction" but most people probably do not need to know in-depth information about the cache as it should "just work". As a user I care about the fact that it is fast...but do I really need to know how we make it fast and how we implemented it? I'd say at 75% of users don't care. That's why I felt that it was reference material.

Reply:
I agree. I think it's a very good call to remove it from the introduction. Most of the people don't care. In fact they should not care. The cache should "just work" transparently. Hence, it's not even something for a more detailed "normal" chapter I think. Because it explains internals of Gradle, but nothing about how I setup or customize my build.

So putting it into a "glossary" or an "advanced topics" section is good.

Reply:
Side note: In fact I moved the "dependency cache" content as a preliminary step when rearranging all the content in the massive chapter. I didn't intent it to live hear with such detail.